envs/quadrotors/quadrotor_params.py

```diff
@@ -19,4 +19,8 @@ def add_quadrotors_env_args(env, parser):
     p.add_argument('--quads_clip_input', default=False, type=str2bool, help='Whether to clip input to ensure it stays relatively small')
     p.add_argument('--quads_effort_reward', default=None, type=float, help='Override default value for effort reward')
     p.add_argument('--quads_episode_duration', default=7.0, type=float, help='Override default value for episode duration')
-    p.add_argument('--quads_num_agents', default=4, type=int, help='Override default value for the number of quadrotors')
\ No newline at end of file
+    p.add_argument('--quads_num_agents', default=4, type=int, help='Override default value for the number of quadrotors')
+    p.add_argument('--quads_collision_reward', default=None, type=float, help='Override default value for quadcol_bin reward')
+    p.add_argument('--quads_settle_reward', default=None, type=float, help='Override default value for quadsettle reward')
```

Comment:
Just to make sure, it will work with default (None) values, and not crash as Sumeet reported?